tools/pip/setup.py

```diff
@@ -66,12 +66,8 @@ def has_ext_modules(self):
 shutil.copy(LIB_PATH[0], os.path.join(CURRENT_DIR, 'mxnet'))
 
 # copy license and notice
-shutil.copy(os.path.join(CURRENT_DIR, 'mxnet-build/LICENSE'),
-            os.path.join(CURRENT_DIR, 'mxnet/LICENSE'))
-shutil.copy(os.path.join(CURRENT_DIR, 'mxnet-build/DISCLAIMER'),
```

Comment:
We do not currently add this DISCLAIMER. Can you add it to the build similarly to https://github.com/apache/incubator-mxnet/pull/14726/files#diff-2c9b07f53d24caa1ac38f1eb9c371118R70?